<launch>

<node name="ld06_lidar" pkg="ld06_lidar" type="ld06_lidar" args="LD06" output="screen" >
    <param name="serial_port" value="/dev/ttyUSB0"/>
    <param name="lidar_frame" value="lidar"/>
    <remap from="/LiDAR/LD06" to="scan"/>
 </node>
</launch>
